src/main/java/com/thevoxelbox/voxelsniper/performer/Performer.java

```diff
@@ -2,14 +2,53 @@
 
 import com.sk89q.worldedit.EditSession;
 import com.sk89q.worldedit.world.block.BlockState;
+import com.thevoxelbox.voxelsniper.performer.property.PerformerProperties;
 import com.thevoxelbox.voxelsniper.sniper.snipe.performer.PerformerSnipe;
 
 public interface Performer {
 
+    /**
+     * Initialize performer.
+     *
+     * @param snipe Snipe
```

Comment:
Eh?

Reply:
Forgot a word, but actually this method initialize data needed by ``perform`` method (local fields, empty lists, etc).